source/extensions/filters/http/admission_control/thread_local_controller.cc

```diff
@@ -17,6 +17,16 @@ ThreadLocalControllerImpl::ThreadLocalControllerImpl(TimeSource& time_source,
                                                      std::chrono::seconds sampling_window)
     : time_source_(time_source), sampling_window_(sampling_window) {}
 
+uint32_t ThreadLocalControllerImpl::averageRps() const {
+  // In the very beginning of sampling, the average rps might be very big, so just skipping from it.
```

Comment:
I don't quite understand what you mean here. Why do you want to do this?

Reply:
The first few requests in the first second may have a short interval, resulting in a big average rps. So skip the calculation of the first second.

Reply:
I see, because you're allowing fractional seconds due to the age being measured in microseconds. 

You can just set the min sample age to 1sec.

Reply:
Sounds reasonable.